library/wizard/src/modules/Wizard.rb

```diff
@@ -92,6 +92,42 @@ def haveFancyUI
       @have_fancy_ui_cache
     end
 
+    # Returns a button box with buttons "Back", "Cancel", "Next"
+    #
+    # @return [Yast::Term] a widget tree representing the buttons
+    def BackCancelNextButtonBox
+      HBox(
+        HWeight(
+          1,
+          ReplacePoint(
+            Id(:rep_help),
+            PushButton(Id(:help), Opt(:key_F1, :helpButton), Label.HelpButton)
+          )
+        ),
+        HStretch(),
+        HWeight(
+          1,
+          ReplacePoint(
+            Id(:rep_back),
+            PushButton(Id(:back), Opt(:key_F8), Label.BackButton)
+          )
+        ),
+        HStretch(),
+        HWeight(
+          1,
+          PushButton(Id(:cancel), Opt(:key_F9, :cancelButton), Label.CancelButton)
+        ),
+        HStretch(),
+        HWeight(
+          1,
+          ReplacePoint(
+            Id(:rep_next),
+            PushButton(Id(:next), Opt(:key_F10, :default), Label.NextButton)
+          )
+        )
+      )
+    end
+
```

Comment:
hmm, I am not much happy with this additional code increase which result that this ruby file has almost 2k lines. I know that it is a bit more work, but is possible to somehow refactor ( in sense of API and functionality) this module to reduce huge duplication here and decrease size?

Reply:
Yes, maybe we can refac that code, but the problem I see is that thd changes here are for SP0, so I prefer a more conservative change here. I would prefer to do such refactorization as part of another PBI, and not for SP0.